## Question 1

### (A).
$\hat{ATE} = \frac{3}{6}-\frac{3}{6} = 0$

### (B).
$ATE = \frac{6}{12} = 0.5$

### (C).
Because Assignment to treatment is not uncorrelated with outcomes,we have selection bias in this case. Therefore, ATE and $\hat{ATE}$ is different.

### (D).
$ATT = \frac{1+1-1}{6} = \frac{1}{6}$

$ATC = \frac{0}{6} = 0$

### (E).
$SB(0) =\frac{2}{6} - \frac{3}{6} = -\frac{1}{6}$

$SB(1) =\frac{3}{6} - \frac{3}{6} = 0$

## Question 2

### (A).

In [2]:
# Load the social pressure experiment results data
SocialPressure = read.csv("D:/Study/Business Analytics/LA/HW6/SocialPressure.csv")
str(SocialPressure)Zz

'data.frame':	344084 obs. of  14 variables:
 $ Vote     : int  0 0 1 0 1 0 1 1 0 1 ...
 $ CivicDuty: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Hawthorne: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Self     : int  0 0 0 0 0 0 0 0 0 0 ...
 $ Neighbors: int  0 0 0 0 0 0 0 0 0 0 ...
 $ hh_size  : int  3 3 4 3 4 4 2 2 4 4 ...
 $ g2000    : int  1 1 0 0 0 0 0 0 0 1 ...
 $ g2002    : int  0 1 1 1 0 0 1 1 1 1 ...
 $ g2004    : int  1 0 1 1 0 0 0 0 0 0 ...
 $ p2000    : int  0 0 0 0 0 1 0 1 0 0 ...
 $ p2002    : int  1 0 0 1 0 0 0 0 0 0 ...
 $ p2004    : int  0 0 1 0 1 1 1 1 1 1 ...
 $ gender   : int  0 0 1 1 0 0 1 1 1 1 ...
 $ yob      : int  1987 1967 1979 1951 1942 1954 1982 1985 1966 1968 ...


In [3]:
# Estimating the causal effect of different treatments
#Control group
Control = subset(subset(subset(subset(SocialPressure,CivicDuty==0),Hawthorne==0),Self==0),Neighbors==0)
str(Control)

'data.frame':	191243 obs. of  14 variables:
 $ Vote     : int  0 0 1 0 1 0 1 1 0 1 ...
 $ CivicDuty: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Hawthorne: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Self     : int  0 0 0 0 0 0 0 0 0 0 ...
 $ Neighbors: int  0 0 0 0 0 0 0 0 0 0 ...
 $ hh_size  : int  3 3 4 3 4 4 2 2 4 4 ...
 $ g2000    : int  1 1 0 0 0 0 0 0 0 1 ...
 $ g2002    : int  0 1 1 1 0 0 1 1 1 1 ...
 $ g2004    : int  1 0 1 1 0 0 0 0 0 0 ...
 $ p2000    : int  0 0 0 0 0 1 0 1 0 0 ...
 $ p2002    : int  1 0 0 1 0 0 0 0 0 0 ...
 $ p2004    : int  0 0 1 0 1 1 1 1 1 1 ...
 $ gender   : int  0 0 1 1 0 0 1 1 1 1 ...
 $ yob      : int  1987 1967 1979 1951 1942 1954 1982 1985 1966 1968 ...


In [4]:
#Treatment Group of Civic Duty
Treatment_CivicDuty = subset(SocialPressure,CivicDuty==1)
str(Treatment_CivicDuty)

'data.frame':	38218 obs. of  14 variables:
 $ Vote     : int  0 1 1 1 0 0 0 0 0 0 ...
 $ CivicDuty: int  1 1 1 1 1 1 1 1 1 1 ...
 $ Hawthorne: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Self     : int  0 0 0 0 0 0 0 0 0 0 ...
 $ Neighbors: int  0 0 0 0 0 0 0 0 0 0 ...
 $ hh_size  : int  2 3 1 3 4 2 2 2 3 3 ...
 $ g2000    : int  1 0 1 0 1 0 0 0 0 0 ...
 $ g2002    : int  0 0 1 0 1 1 0 1 0 0 ...
 $ g2004    : int  0 1 0 0 0 0 1 0 0 0 ...
 $ p2000    : int  1 0 1 0 0 0 0 1 0 0 ...
 $ p2002    : int  0 0 0 0 0 1 1 0 1 0 ...
 $ p2004    : int  1 0 0 1 1 0 0 1 0 1 ...
 $ gender   : int  1 0 0 1 0 0 1 0 1 0 ...
 $ yob      : int  1964 1940 1937 1972 1980 1955 1953 1958 1956 1973 ...


In [5]:
#Treatment Group of Hawthorne
Treatment_Hawthorne = subset(SocialPressure,Hawthorne==1)
str(Treatment_Hawthorne)

'data.frame':	38204 obs. of  14 variables:
 $ Vote     : int  1 1 0 0 0 0 1 0 0 0 ...
 $ CivicDuty: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Hawthorne: int  1 1 1 1 1 1 1 1 1 1 ...
 $ Self     : int  0 0 0 0 0 0 0 0 0 0 ...
 $ Neighbors: int  0 0 0 0 0 0 0 0 0 0 ...
 $ hh_size  : int  2 1 3 3 4 3 1 2 3 3 ...
 $ g2000    : int  1 0 0 1 0 0 0 0 1 0 ...
 $ g2002    : int  1 1 0 0 1 1 0 1 1 1 ...
 $ g2004    : int  1 1 1 0 1 1 0 0 1 0 ...
 $ p2000    : int  0 0 1 0 1 0 1 1 0 1 ...
 $ p2002    : int  1 0 0 1 0 0 0 0 1 0 ...
 $ p2004    : int  0 1 0 1 0 1 1 0 0 0 ...
 $ gender   : int  1 0 0 1 1 0 0 1 0 1 ...
 $ yob      : int  1986 1945 1941 1976 1942 1944 1975 1949 1952 1940 ...


In [6]:
#Treatment Group of Self
Treatment_Self = subset(SocialPressure,Self==1)
str(Treatment_Self)

'data.frame':	38218 obs. of  14 variables:
 $ Vote     : int  0 0 1 0 1 1 1 0 0 1 ...
 $ CivicDuty: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Hawthorne: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Self     : int  1 1 1 1 1 1 1 1 1 1 ...
 $ Neighbors: int  0 0 0 0 0 0 0 0 0 0 ...
 $ hh_size  : int  4 3 2 3 2 2 3 2 2 3 ...
 $ g2000    : int  1 0 0 1 0 0 1 1 0 1 ...
 $ g2002    : int  1 0 0 0 0 0 0 1 0 0 ...
 $ g2004    : int  0 0 0 1 1 1 0 1 0 0 ...
 $ p2000    : int  0 0 0 1 0 1 0 0 0 0 ...
 $ p2002    : int  0 1 0 0 0 0 1 0 1 0 ...
 $ p2004    : int  1 0 0 0 0 0 1 0 0 0 ...
 $ gender   : int  0 0 1 1 1 1 1 1 0 1 ...
 $ yob      : int  1976 1954 1966 1969 1941 1988 1942 1938 1945 1981 ...


In [7]:
#Treatment Group of Neighbors
Treatment_Neighbors = subset(SocialPressure,Neighbors==1)
str(Treatment_Neighbors)

'data.frame':	38201 obs. of  14 variables:
 $ Vote     : int  1 0 1 1 1 0 0 0 0 0 ...
 $ CivicDuty: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Hawthorne: int  0 0 0 0 0 0 0 0 0 0 ...
 $ Self     : int  0 0 0 0 0 0 0 0 0 0 ...
 $ Neighbors: int  1 1 1 1 1 1 1 1 1 1 ...
 $ hh_size  : int  2 2 2 3 2 1 2 3 2 3 ...
 $ g2000    : int  0 1 0 0 0 1 1 1 1 0 ...
 $ g2002    : int  0 1 0 0 1 0 0 0 1 1 ...
 $ g2004    : int  1 0 0 0 0 0 0 0 1 0 ...
 $ p2000    : int  1 0 1 1 1 0 0 1 0 1 ...
 $ p2002    : int  0 0 0 0 0 0 0 1 0 0 ...
 $ p2004    : int  1 0 0 0 0 1 0 0 0 0 ...
 $ gender   : int  1 0 0 1 1 0 1 1 0 1 ...
 $ yob      : int  1979 1950 1939 1956 1979 1986 1937 1949 1965 1948 ...


In [ ]:
## Use Regression to do balance check
BalanceCheck1 = lm(CivicDuty~hh_size+g2000+g2002+g2004+p2000+p2002+p2004+gender+yob,data = SocialPressure)
summary(BalanceCheck1)

BalanceCheck2 = lm(Hawthorne~hh_size+g2000+g2002+g2004+p2000+p2002+p2004+gender+yob,data = SocialPressure)
summary(BalanceCheck2)

BalanceCheck3 = lm(Self~hh_size+g2000+g2002+g2004+p2000+p2002+p2004+gender+yob,data = SocialPressure)
summary(BalanceCheck3)

BalanceCheck4 = lm(Neighbors~hh_size+g2000+g2002+g2004+p2000+p2002+p2004+gender+yob,data = SocialPressure)
summary(BalanceCheck4)

We check the t-values of the independent variables in the regressions. We found that in most of the cases the variables are not statistically significant, so we consider the sample to be balanced.

## Question 3

In [1]:
Corruption = read.csv("D:/Study/Business Analytics/LA/HW6/CorruptionExperiment.csv")
str(Corruption)

'data.frame':	472 obs. of  7 variables:
 $ id          : int  1 2 3 4 5 6 7 8 9 10 ...
 $ treat.invite: int  0 1 1 1 0 1 0 1 1 0 ...
 $ pct.missing : num  0.3853 -0.0957 0.1477 -0.1826 -0.293 ...
 $ head.edu    : int  6 14 12 9 9 9 12 16 12 12 ...
 $ mosques     : num  0.908 1.067 0.712 0.949 1.623 ...
 $ pct.poor    : num  0.4 0.186 0.4 0.438 0.313 ...
 $ total.budget: num  40.6 69.3 41.1 17.1 72.1 ...


## (A).

In [2]:
BalanceCheck = lm(treat.invite~head.edu +mosques+pct.poor+total.budget,data = Corruption)
summary(BalanceCheck)


Call:
lm(formula = treat.invite ~ head.edu + mosques + pct.poor + total.budget, 
    data = Corruption)

Residuals:
    Min      1Q  Median      3Q     Max 
-0.7075 -0.6474  0.3263  0.3449  0.4480 

Coefficients:
               Estimate Std. Error t value Pr(>|t|)    
(Intercept)   6.767e-01  1.172e-01   5.772 1.43e-08 ***
head.edu     -8.972e-04  8.099e-03  -0.111    0.912    
mosques      -1.871e-02  2.646e-02  -0.707    0.480    
pct.poor      5.744e-02  1.044e-01   0.550    0.582    
total.budget -4.698e-05  4.013e-04  -0.117    0.907    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 0.4762 on 467 degrees of freedom
Multiple R-squared:  0.00163,	Adjusted R-squared:  -0.006921 
F-statistic: 0.1906 on 4 and 467 DF,  p-value: 0.9433


The independent variables are not statistically significant, so the sample can be considered as balanced.

## (B).

In [3]:
# Calculate the estimated ATE and the standard deviation of ATE

Control = subset(Corruption,treat.invite==0)
Treatment = subset(Corruption,treat.invite==1)

n1 = nrow(Treatment)
n0 = nrow(Control)

ATE = mean(Treatment$pct.missing)-mean(Control$pct.missing)
ATE.STD=sqrt(sd(Treatment$pct.missing)^2/n1+sd(Control$pct.missing)^2/n0)

UCB_95 = ATE+1.96*ATE.STD
LCB_95 = ATE-1.96*ATE.STD


ATE
UCB_95
LCB_95

[1] -0.02494953

[1] 0.03992684

[1] -0.0898259

In [4]:
# And usinng regression
Regression1 = lm(pct.missing~treat.invite,data=Corruption)
summary(Regression1)


Call:
lm(formula = pct.missing ~ treat.invite, data = Corruption)

Residuals:
     Min       1Q   Median       3Q      Max 
-1.33064 -0.21249 -0.01284  0.18281  1.42154 

Coefficients:
             Estimate Std. Error t value Pr(>|t|)    
(Intercept)   0.25277    0.02716   9.306   <2e-16 ***
treat.invite -0.02495    0.03346  -0.746    0.456    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 0.3447 on 470 degrees of freedom
Multiple R-squared:  0.001181,	Adjusted R-squared:  -0.0009438 
F-statistic: 0.5559 on 1 and 470 DF,  p-value: 0.4563


The treatment is not statistically significant with 95% Confidence Interval

## (C）

In [2]:
# And usinng regression
Regression1 = lm(pct.missing~treat.invite+head.edu +mosques+pct.poor+total.budget,data=Corruption)
summary(Regression1)


Call:
lm(formula = pct.missing ~ treat.invite + head.edu + mosques + 
    pct.poor + total.budget, data = Corruption)

Residuals:
     Min       1Q   Median       3Q      Max 
-1.28605 -0.21411 -0.01291  0.18932  1.42530 

Coefficients:
               Estimate Std. Error t value Pr(>|t|)    
(Intercept)   0.3904455  0.0869471   4.491 8.96e-06 ***
treat.invite -0.0264183  0.0331558  -0.797   0.4260    
head.edu     -0.0055082  0.0058032  -0.949   0.3430    
mosques      -0.0481914  0.0189702  -2.540   0.0114 *  
pct.poor     -0.1177125  0.0747921  -1.574   0.1162    
total.budget  0.0005307  0.0002875   1.846   0.0655 .  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 0.3412 on 466 degrees of freedom
Multiple R-squared:  0.0294,	Adjusted R-squared:  0.01898 
F-statistic: 2.823 on 5 and 466 DF,  p-value: 0.01594


The estimated coefficient is -0.0264183. As we see from question 3(A), the ATE estimate is -0.0249495273729704.
These are not significantly.